src/homework/day8/collectionsforclasses/SandList.java

```diff
@@ -0,0 +1,69 @@
+package homework.day8.collectionsforclasses;
+
+import homework.day8.absclasses.Sand;
+
+import java.util.*;
+
+public class SandList {
+    static Random random = new Random();
+
+    public static void main(String[] args) {
+        List<Sand> sandbox = new ArrayList<>();
+        sandbox.add(new Sand(2, "Речной"));
+        sandbox.add(new Sand(4, "Речной"));
+        sandbox.add(new Sand(2, "Карьерный"));
+        sandbox.add(new Sand(7, "Речной"));
+        printSandWeight(sandbox);
+        printSandName(sandbox);
+        Map<Integer, Sand> sandMap = new HashMap<>();
+        fillMapWithObjects(sandbox, sandMap);
+        printMapKey(sandMap);
+        printMapValue(sandMap);
+        printKeyMapPair(sandMap);
+
+    }
+
+    public static void printSandName(List<Sand> anySandList) {
+        for (Sand sand : anySandList) {
+            System.out.print(sand.getName() + " ");
+        }
+        System.out.println();
+    }
+
+    public static void printSandWeight(List<Sand> anySandList) {
+        for (Sand sand : anySandList) {
+            System.out.print(sand.getWeight() + " ");
+        }
+        System.out.println();
+    }
+
+    public static Map<Integer, Sand> fillMapWithObjects(List<Sand> sandList, Map<Integer, Sand> sandMap) {
+        for (Sand sand : sandList) {
+            sandMap.put(random.nextInt(1, 10000), sand);
```

Comment:
using a randomizer there is a chance that the number will not be unique, you've choose for this huge range, to avoid this situation, but it's point to think about

Reply:
method outdated, not exists in SandList.java